domains/authorization/Authorization.API/API/Authorization/Controllers/AuthorizationController.cs

```diff
@@ -10,6 +11,7 @@ namespace API.Authorization.Controllers;
 
 [ApiController]
 [ApiVersion(ApiVersions.Version20230101)]
+[Authorize(Policy = Policy.B2CPolicy)]
```

Comment:
I believe these specific two endpoints should use policy _B2CCustomPolicyClientPolicy_.

They should only be called from B2C from our custom policies.

Reply:
fixed